packages/ai/src/ai/modules/task/commands/cmd_account.py

```
@@ -0,0 +1,1243 @@
+
+        Returns:
+            DAP success response with an empty body.
+
+        Raises:
+            ValueError: If ``teamId`` is missing, the user does not exist, or the
+                        user is not a member of the specified team.
+        """
+        # Validate that the caller supplied a team ID — empty string is rejected.
+        team_id = args.get('teamId', '')
+        if not team_id:
+            raise ValueError('teamId is required')
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)